tck/src/main/java/ee/jakarta/tck/data/framework/read/only/City.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package ee.jakarta.tck.data.framework.read.only;
+
+@jakarta.nosql.Embeddable
+@jakarta.persistence.Embeddable
+public class City {
```

Comment:
I believe that both could be record right?
I mean, on Jakarta NoSQL we support it and on jakarta Persistence we have it since Jakarta EE 11.

it is up to you.

Reply:
Yes, it would be valid to use a record for the embeddable in JPA.  Either way is fine with me, so I can switch it.

Reply:
Actually, I'm going to wait on switching the embeddable to record because EclipseLink still has a known bug with records as embeddables that needs to be fixed and I'd like to still be able to run these tests to verify they are valid. I'll leave it as as normal class for now, but I'm willing to switch it to record in the future after the EclipseLink bug is fixed.

Reply:
It is up to you ok, I am fine with both.